### ISSUE TITLE
something is wrong

### ISSUE THREAD

**Issue by BlumeCoding** [gigajew/Mandark#2]
doesnt seems to work when it injects i see windows give me werfaul.exe

hab kp vlt net framework ?

**Comment by gigajew** [comment]
What binaries are you using? Did you compile to 64-bit? Are you injecting into a 64-bit process?

**Comment by BlumeCoding** [comment]
yes im compiling it in 64 bit using amd cpu and also injecting it into an 64 bit process

**Comment by gigajew** [comment]
Tried running as administrator?

**Comment by BlumeCoding** [comment]
Yes

gigajew <notifications@github.com> schrieb am Di. 19. Jan. 2021 um 19:47:
